shelley/chain-and-ledger/formal-spec/ledger-spec.tex

```diff
@@ -223,6 +223,7 @@
 \include{ledger}
 \include{epoch}
 \include{chain}
+\include{sts-overview}
```

Comment:
did you mean to add a `sts-overview.tex` file that displayed the new image?

Reply:
yes, sorry, forgot that